geode-core/src/main/java/org/apache/geode/distributed/AbstractLauncher.java

```diff
@@ -156,7 +166,14 @@ protected static Properties loadGemFireProperties(final URL url) {
 
     if (url != null) {
       try {
-        properties.load(new FileReader(new File(url.toURI())));
+        File propertyFile = new File(url.toURI());
```

Comment:
This url is resolved by using gemfire.properties as default already. See DistributedSystem.getPropertiesFileURL(). So if gemfirePropertiesFile is not defined as a system property, it will use "gemfire.properties" first. We need to add another constant as GEODE_PREFIX in distribution config if we can not totally replace the GEMFIRE_PREFIX.